### PR TITLE
Prep for alpha release

### PR DESCRIPTION
**Pull request by frreiss**
This PR includes changes to prepare for an alpha release. I have been using the source branch for this PR to test release tooling. Current release testing results:
* Github pages web site: https://frreiss.github.io/tep-fred/
* Github repo page: https://github.com/frreiss/tep-fred/tree/branch-alpha
* Package: https://test.pypi.org/project/text-extensions-for-pandas/0.1a4/

Main changes in this PR:
* Created a (stub) web site for Github pages under `docs`  (Github pages requires the web site to be either at the root of the repo or in a subdirectory called `docs`)
* Moved the API docs that used to be at `docs` to `api_docs`
* Fixed some minor problems with project metadata
* Rewrote most of `README.md`
* Created a readme for the PyPI page: `package.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,114 +1,131 @@
...
-### Contents of this repository
+## Contents of this repository
 
 * **`text_extensions_for_pandas`**: Source code for the `text_extensions_for_pandas` module.
-* **notebooks**: demo notebooks
-* **resources**: various input files used by the demo notebooks 
-* **env.sh**: Script to create an conda environment `pd` capable of running the notebooks in this directory
-
-### Instructions to run a demo notebook
+* **env.sh**: Script to create an conda environment `pd` capable of running the notebooks and test cases in this project
```
Comment (BryanCutler) [inline README.md:73]: an -> a

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!-- Front page of web site -->
+
+# Web page test
+
+This should be the main page of the Github Pages site.
```
Comment (BryanCutler) [inline docs/README.md:5]: Was this meant to be left in?

**Review by BryanCutler — APPROVED** [review]
LGTM, just noticed 2 minor things

File: README.md (hunk 1 above)
Comment (frreiss) [reply README.md:73]: Fixed now, thanks.

File: docs/README.md (hunk 2 above)
Comment (frreiss) [reply docs/README.md:5]: The web page is just a stub at the moment. I need to spend some quality time with my text editor creating some content.

**Comment by frreiss** [comment]
Merging these changes to master as soon as CI finishes.

**Pull request merged**